stdlib/typing.pyi

```diff
@@ -524,6 +527,12 @@ if sys.version_info >= (3, 8):
 if sys.version_info >= (3, 11):
     Self: _SpecialForm
     Never: _SpecialForm = ...
+    Unpack: _SpecialForm
+
+    class TypeVarTuple:
+        __name__: str
```

Comment:
Anticipating python/cpython#31954.

Reply:
Should we wait until that's merged before merging this, or do you reckon we're good to go?

Reply:
Better to wait I think.

Reply:
(Converting this to a draft for now so it doesn't get accidentally merged in the meantime)

Reply:
I merged the CPython change

Reply:
And I merged this one :)